### PR TITLE
Feature mes 2265 switch welsh test to english

### PR DESCRIPTION
**Pull request by MarkPatt89**
## Description and relevant Jira numbers
- Added 'ConductedLanguageComponent' to display Welsh/English toggle on 'Communication Capture Page'
- Dispatch actions created to handle state management
## Pull Request checklist:

- [x] [WIP] tag removed from PR title
- [x] PR has an understandable description

## Git feature branch checklist:

- [x] branch name comply with our branching strategy
- [x] git branch contains relevant JIRA ticket number
- [x] branch rebased against the latest develop
- [ ] commits are squashed

## Sign off process checklist:

- [x] Code has been tested manually
- [x] Tests are passing
- [x] PR link added to JIRA
- [x] Reviewers selected in Github
- [ ] Any new reusable component/widget added to component library on Confluence

- [x] Screenshot(s) captured on the physical device (if applicable)
- [ ] Tested by QA
- [ ] PO's approval

Screenshot 2019-05-27 at 16 55 33

Screenshot 2019-05-27 at 16 55 37

Screenshot 2019-05-27 at 16 55 42

### CODE REVIEW

File: src/components/conducted-language/conducted-language.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<ion-row *ngIf="shouldRender">
+  <ion-col></ion-col>
```
Comment (johnmorleybjss) [inline src/components/conducted-language/conducted-language.html]: Do we need this first column?

File: src/components/conducted-language/conducted-language.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  englishIsSelected: boolean;
+
+  @Input()
+  shouldRender: boolean;
+
+  @Output()
+  welshTextSelect = new EventEmitter<boolean>();
+
+  @Output()
+  englishTextSelect = new EventEmitter<boolean>();
```
Comment (johnmorleybjss) [inline src/components/conducted-language/conducted-language.ts]: Couldn't these be `EventEmitter<void>`?

Comment (MarkPatt89) [reply src/components/conducted-language/conducted-language.ts]: Yes, they could. I'll change it now.

File: src/components/conducted-language/conducted-language.html (hunk 1 above)
Comment (MarkPatt89) [reply src/components/conducted-language/conducted-language.html]: No, it's redundant I guess. It was to spare a huge amount of padding. I will remove. 👍

File: src/modules/tests/communication-preferences/communication-preferences.selector.ts [hunk 3]
```diff
@@ -6,3 +6,6 @@ export const getCommunicationPreferenceUpdatedEmail
 
 export const getCommunicationPreferenceType
   = (communicationPreferences: CommunicationPreferences) => get(communicationPreferences, 'communicationMethod', '');
+
+export const getConductedLanguage
+  = (communicationPreferences: CommunicationPreferences) => get(communicationPreferences, 'conductedLanguage', '');
```
Comment (johnmorleybjss) [inline src/modules/tests/communication-preferences/communication-preferences.selector.ts]: Do we really need a `get` here? Is there a case we need to guard against null? If there is, for 1 level deep I might be inclined to do it manually.

Comment (MarkPatt89) [reply src/modules/tests/communication-preferences/communication-preferences.selector.ts]: Done

**Pull request merged**